openshift/ci-operator/source-image/Dockerfile

```diff
@@ -4,5 +4,6 @@ COPY oc /usr/bin/oc
 COPY --from=registry.ci.openshift.org/openshift/knative-v1.0.1:knative-serving-src /go/src/knative.dev/serving/ /go/src/knative.dev/serving/
 COPY --from=registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-src /go/src/knative.dev/eventing/ /go/src/knative.dev/eventing/
 COPY --from=registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-kafka-src /go/src/knative.dev/eventing-kafka/ /go/src/knative.dev/eventing-kafka/
+COPY --from=registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-kafka-broker-src /go/src/knative.dev/eventing-kafka-broker/ /go/src/knative.dev/eventing-kafka-broker/
 
 RUN chmod g+w /go/src/knative.dev/serving/ && chmod g+w /go/src/knative.dev/eventing/ && chmod g+w /go/src/knative.dev/eventing-kafka/
```

Comment:
```suggestion
RUN chmod g+w /go/src/knative.dev/serving/ && chmod g+w /go/src/knative.dev/eventing/ && chmod g+w /go/src/knative.dev/eventing-kafka/ && chmod g+w /go/src/knative.dev/eventing-kafka-broker/
```

Reply:
This is actually not necessary. We only need to change it in the template below and re-generate.

Reply:
I think that was changed, when doing the generate ? not sure what you mean